README.md

```diff
@@ -166,12 +168,17 @@ Now you'll need to do steps (2) everytime you make a changes and want to build t
 
 The Makefile has following options, that are useful for developers & users. Default action is `compose-build`.
 
+**NOTE:** When you are managing your custom images, please change `DOCKER_USERNAME` with your docker account by export environment variable or passing directly to the make command. For example, if you want to publish your image, use `make publish DOCKER_USERNAME=<your-docker-username>`
```

Comment:
Well, should I just mentioned it in `Build (Development)` section? I'm afraid that users will not notice this because it is an important notice whether it's in Deployment, Build (Development), and even Makefile. If users want to deploy their custom images, then they should set `DOCKER_USERNAME` and it's appropriate tag according to their own image, then run `docker-compose up`. That is important and if the users forgot, then docker-compose will use openwisp latest images instead. Also I do agree with that example, but then if you want to use multiple `make` calls, you have to export the variable instead of repeating yourself adding that `DOCKER_USERNAME=*`, that's why I mentioned it in the Makefile section.
So which place do you think the better for this notice?

Reply:
Users will only need to deploy their own images when they want to change something in the image, i.e build the image.
I can't think of a senario where I would want a seperate image to manage for myself but I wouldn't want to change / build the images, can you share what senario you are thinking? :smile:

Reply:
> That is important and if the users forgot, then docker-compose will use openwisp latest images instead. Also I do agree with that example, but then if you want to use multiple make calls, you have to export the variable instead of repeating yourself adding that DOCKER_USERNAME=*

Well, it's technically a requirement only for `make publish`, right? So we should mention it in relation to that only in my opinion, if someone has to use `make` multiple times without mentioning the environment variable, I guess they will have to learn environment variables! :smile:

Reply:
I think it should be mentioned at the first part of the workflow in which it will be needed.  If it's not needed until you actually push out images then I would not mention it early on.  I often build the containers to test development but never have a need to push them to dockerhub.